packages/engine/src/scene/behaviors/createBoxCollider.ts

```diff
@@ -1,11 +1,12 @@
+import { Quaternion, Vector3 } from '@etherealjs/core';
```

Comment:
Let’s not pull these types from ethereal :)

Just grab them directly from THREE